### PR TITLE
[CP-2995] Contacts Panel added (wrapper)

### PR DESCRIPTION
**Pull request by dkarski**
JIRA Reference: [CP-2995]

### :memo: Description ️

-

### :muscle: Checklist before requesting a review - nice to have

- getState function in async thunk actions is correctly typed
- redux selectors are used in components / prop drilling is reduce

### :exclamation: Checklist before merging a pull request

- change went through the QA process
- translations are updated in dedicated application
- [CHANGELOG.md](./CHANGELOG.md) is updated

[CP-2995]: https://appnroll.atlassian.net/browse/CP-2995?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts [hunk 1]
```diff
@@ -16,7 +16,70 @@ const view: View = {
...
+    layout: {
+      gridLayout: {
+        rows: [],
+        columns: ["280px", "344px"],
+        justifyContent: "space-between",
+      },
+    },
+  },
+  contactsSearch: {
+    component: "form",
```
Comment (mkurczewski) [inline libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts:33]: Form should be above everything else but under loader

File: libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts [hunk 2]
```diff
@@ -16,7 +16,70 @@ const view: View = {
...
+    config: {
+      formOptions: {
+        defaultValues: {
+          searchedContact: undefined,
+        },
+      },
+    },
+    childrenKeys: ["contactsSearchWrapper"],
```
Comment (mkurczewski) [inline libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts:41]: `contactsSearchWrapper` is not necessary, can be replaced by `contactsSearchInput`

File: libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts [hunk 3]
```diff
@@ -16,7 +16,70 @@ const view: View = {
...
+  },
+  contactsSearchInput: {
+    component: "form.searchInput",
+    config: {
+      label: "Search all contacts",
+      name: "searchedContact",
+    },
+  },
+  contactsButtonActions: {
+    component: "modal.buttons",
```
Comment (mkurczewski) [inline libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts:55]: Better to use `block-plain` with proper layout (flexLayout with direction and gap) or (gridLayout with columns [1fr,1fr] and gap)

File: libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts (hunk 2 above)
Comment (dkarski) [reply libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts]: https://github.com/mudita/mudita-center/pull/2078/commits/ff1765b598b7425aad42b18003c5cdf4603df84f

File: libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts (hunk 1 above)
Comment (dkarski) [reply libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts:22]: https://github.com/mudita/mudita-center/pull/2078/commits/ff1765b598b7425aad42b18003c5cdf4603df84f

File: libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts [hunk 4]
```diff
@@ -16,7 +16,70 @@ const view: View = {
         columns: ["1fr"],
       },
     },
-    childrenKeys: ["importContactsButton", "contactsLoader"],
+    childrenKeys: ["contactsPanel", "contactsLoader"],
```
[the remark this answers is not in the document]
Comment (dkarski) [reply libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts]: https://github.com/mudita/mudita-center/pull/2078/commits/ff1765b598b7425aad42b18003c5cdf4603df84f

**Review by mkurczewski — APPROVED** [review]

**Pull request merged**